tools/marketplace/main.go

```diff
@@ -45,7 +45,7 @@ func main() {
 		fmt.Printf("csv: %v\n", helper.GetCSVName(csv))
 	}
 	for _, crd := range bh.CRDs {
-		fmt.Printf("crd: %v %v\n", crd.Name, crd.Spec.Version)
```

Comment:
Here `Version` is a string and `Versions` is an array of Versions (structs), I'm not sure you want to print out the whole struct, also this probably belongs in a separate PR or at least a separate commit otherwise backporting this if needed might be awkward, wdyt?

Reply:
I changed it because my IDE was showing `crd.Spec.Version` as an invalid field. When I followed up, I noticed that the [ spec](https://github.com/kubernetes/apiextensions-apiserver/blob/master/pkg/apis/apiextensions/types.go#L67) does not define a `Version` field in the `CustomResourceDefinitionSpec` struct. 
```
crd.Spec.Version undefined (type v1.CustomResourceDefinitionSpec has no field or method Version)
```
This change is not related to the PR, but more of a cleanup. I can remove it if you think it should not be there.